### PR TITLE
Add flash messages to talk form

### PR DESCRIPTION
**Pull request by vlatkapavisic**

### CODE REVIEW

**Comment by vlatkapavisic** [comment]
@abelards who can I request the review from?

**Comment by ylecuyer** [comment]
I was going to create the same PR :+1: this one needs to be merged @abelards

File: app/assets/stylesheets/base.scss [hunk 1]
```diff
@@ -144,3 +144,10 @@ body > footer p {
     display: block;
   }
 }
+
+.notice {
+  background-color: #5d202657;
```
Comment (ylecuyer) [inline app/assets/stylesheets/base.scss]: Which color is this?

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -15,10 +15,9 @@ html lang="fr"
   body
     = render 'shared/navbar'
     .main-content
-      - if notice
-        p.notice = notice
-      - if alert
-        p.alert = alert
+      .notice
+        - if flash[:notice]
```
Comment (ylecuyer) [inline app/views/layouts/application.html.slim]: Would be better with something like : 

```
flash.each do |key, value|
```

File: app/assets/stylesheets/base.scss (hunk 1 above)
Comment (vlatkapavisic) [reply app/assets/stylesheets/base.scss]: A variation of a dark red website's theme color.

**Pull request closed without merging**